components/reusable-components/button/button.module.scss

```diff
@@ -1,18 +1,56 @@
-.button {
-  padding: 10px 20px;
+.primaryButton {
+  display: flex;
+  gap: 8px;
+  background-color: var(--primary-btn-color);
+  color: var(--text-color);
+  border-radius: 8px;
+  padding: 8px 24px 8px 24px;
   font-size: 16px;
-  background-color: #007bff;
+  font-weight: 700;
+  line-height: 20px;
+  cursor: pointer;
+  transition: background-color 0.3s ease-in-out;
+
+  &:hover {
+    background-color: var(--primary-btn-hover-color);
+  }
+
+  &:active {
+    background-color: var(--primary-btn-active-color);
+  }
+
+  &:focus {
+    outline: none;
+  }
+}
+
+.secondaryButton {
+  display: flex;
+  gap: 8px;
+  background-color: transparent;
   color: var(--text-color);
-  border: none;
-  border-radius: 5px;
+  border: 1px solid var(--primary-btn-color);
+  border-radius: 8px;
+  padding: 8px 24px 8px 24px;
```

Comment:
same here, we can maybe have one class for the styles that both buttons share, and the primary button would have the .primaryButton for the styles that are unique to the primaryButton. same for secondary button